sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -605,6 +620,7 @@ def test_oob_multilcass_iris():
     score = clf.score(iris.data, iris.target)
     assert score > 0.9
     assert clf.oob_improvement_.shape[0] == clf.n_estimators
+    assert clf.oob_scores_.shape[0] == clf.n_estimators
```

Comment:
Ditto: I think this assertion (or even test) needs to be updated to pass when early stopping is used.